app/lib.js

```diff
@@ -64,11 +64,25 @@ function extractVaultFromFile (data) {
     // attempt 4: chromium 000006.log on MacOS
     // this variant also contains a 'keyMetadata' key in the vault, which should be
     // a nested object.
-    const matches = data.match(/KeyringController":(\{"vault":".*=\\"\}"\})/);
+    const matches = data.match(/KeyringController":(\{"vault":".*?=\\"\}"\})/);
```

Comment:
The lazy (`?`) operator here is important, because there are instances of `.log` files that present multiple occurrences of this, and in these cases if we don't use the lazy operator we'll match the entire text between the start of the first occurrence and the end of the last one. We are instead only interested in the first occurrence.